core/emisc.src/1

```diff
@@ -30,6 +30,10 @@ function mount.blk() {
 	esac
 }
 
+rm -rf /dev/gearblock
```

Comment:
BTW, does this really help? Have you presonally faced an issue or seen anyone who could have benefited from this workaround?